mobile/android/android-device-file-system.ts

```diff
@@ -20,17 +20,17 @@ export class AndroidDeviceFileSystem implements Mobile.IDeviceFileSystem {
 	public putFile(localFilePath: string, deviceFilePath: string): IFuture<void> {
 		return (() => { }).future<void>()();
 	}
-	
+
 	public transferFiles(appIdentifier: string, localToDevicePaths: Mobile.ILocalToDevicePathData[]): IFuture<void> {
 		return (() => {
 			_(localToDevicePaths)
 				.filter(localToDevicePathData => this.$fs.getFsStats(localToDevicePathData.getLocalPath()).wait().isFile())
-				.each(localToDevicePathData => this.adb.executeCommand(`push "${localToDevicePathData.getLocalPath()}" "${localToDevicePathData.getDevicePath()}"`).wait())
+				.each(localToDevicePathData => this.adb.executeCommand(["push", `${localToDevicePathData.getLocalPath()}`, `${localToDevicePathData.getDevicePath()}`]).wait())
```

Comment:
we have to add "" around `localToDevicePathData.getLocalPath()` and `localToDevicePathData.getDevicePath()`